packages/cqrs/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ description: >-
   queries and commands.
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: ">=2.12.0 <3.0.0"
```

Comment:
Afaik we are sticking to single quotes

Reply:
VSCode changed it for me, sorry.

Looks like [Flutter prefers double quotes](https://docs.flutter.dev/development/packages-and-plugins/developing-packages#plugin-platforms) and [Dart prefers single quotes](https://dart.dev/tools/pub/pubspec#sdk-constraints) (or it might be just a coincidence and there's no consensus over this, so let's not bikeshed).